src/FluentNHibernate/Conventions/Inspections/JoinedSubclassInspector.cs

```diff
@@ -104,6 +104,11 @@ public bool LazyLoad
             get { return mapping.Lazy; }
         }
 
+        public string Schema

```

Comment:
I hesitated over using explicit implementation (`string IJoinedSubclassInspector.Schema`) instead, but went for direct implementation.